rust/tests/ledger/token_ledger.rs

```diff
@@ -170,7 +176,14 @@ async fn check_token_accounts() -> anyhow::Result<()> {
                 ..Default::default()
             };
 
-        assert_eq!(mina_account.to_owned().display(), expect)
+        assert_eq!(*mina_account, expect);
+        assert_eq!(
+            mina_account.clone().display(),
+            Account {
+                balance: 0.into(),
```

Comment:
`0.into()` or `(1e9 as u64).into()`

Reply:
- `0` for "display"
- `1e9` in the account since it is non-genesis